tests/data/sample_file.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0"?>

```

Comment:
So I guess to create this XML you took a real XLM file from somewhere (?) and modified some of the entries? Or is this an actual template?

Reply:
Yes this is real XML file that was modified. The structure was found on https://jats.nlm.nih.gov/archiving/.

Reply:
Cool. Thanks!